### PR TITLE
Administrative Consideration

### PR DESCRIPTION
**Pull request by ayoumans**
As I was going through, please consider some administrative changes.

### CODE REVIEW

File: core/standard/clause_6_overview.adoc [hunk 1]
```diff
@@ -151,19 +151,19 @@ This standard can be referenced by other standards providing resources that that
 * _OGC API - Processes - Part 3: Workflows and Chaining_ provides a mechanism to trigger localized processing workflows as a result of retrieving maps (for a specific area and resolution of interest).
 
 This document is the first part of a series of _OGC API - Maps_ "parts" that use the core and extensions model.
-It is foreseen that future parts will specify other extensions, such as how to get information of a point in a map.
+It is foreseen that future parts will specify other extensions, such as how to get information on a point in a map.
```
Comment (jerstlouis) [inline core/standard/clause_6_overview.adoc:154]: The intent here really is "of", as in information for a particular geospatial position. Would `for` be better?

File: core/standard/clause_6_overview.adoc [hunk 2]
```diff
@@ -180,7 +180,7 @@ NOTE: Even if this document does not provide a direct GetFeatureInfo equivalent,
 
 === Using this standard independently
 
-Although this standard is designed as a building block that can be leveraged by other standards adding precisions about specific types of data available as maps
+Although this standard is designed as a building block that can be leveraged by other standards by adding precision about specific types of data available as maps
```
Comment (jerstlouis) [inline core/standard/clause_6_overview.adoc:183]: I'm not sure we even want this text in this section. This seems to be carried over from OGC API - Tiles where it had relevance there, but do we really foresee other standard for maps of specific data types?

File: core/standard/clause_2_conformance.adoc [hunk 3]
```diff
@@ -65,7 +65,7 @@ NOTE: Combining this conformance class with the _spatial subsetting by center po
 (not `width` and `height`, which take on a subsetting role).
 
 *<<rc_display-resolution>>* (http://www.opengis.net/spec/ogcapi-maps-1/1.0/req/display-resolution)
-The _display resolution_ conformance class allows to specify the resolution of the display (the size of a pixel in millimeters) using the
+The _display resolution_ conformance class allows the specification the resolution of the display (the size of a pixel in millimeters) using the
```
[the remark this answers is not in the document]
Comment (ayoumans) [reply core/standard/clause_2_conformance.adoc:68]: Ah, yes "of"...that is to problem of me doing this...I don't do changes enough to get good at it or do it efficiently...

File: core/standard/clause_6_overview.adoc (hunk 1 above)
Comment (ayoumans) [reply core/standard/clause_6_overview.adoc:154]: Yes, "for".

File: core/standard/clause_6_overview.adoc (hunk 2 above)
Comment (ayoumans) [reply core/standard/clause_6_overview.adoc:183]: It needs something.....I was not sure what was envisioned.

File: core/standard/clause_6_overview.adoc [hunk 4]
```diff
@@ -211,7 +211,7 @@ for other map encodings).
 ==== Limitations
 
 Although this standard can be used by itself, other OGC API standards or draft specifications may provide additional capabilities and specify additional
-normative requirements describing how to retrieve specific map content, or allowing to describe in greater detail the domain or the observed or
+normative requirements describing how to retrieve specific map content, or allowing the description of greater detail the domain or the observed or
```
Comment (jerstlouis) [inline core/standard/clause_6_overview.adoc:214]: _the description of greater detail the_

This introduces a grammar issue.

_the description in greater detail of the domain..._ perhaps?

File: core/standard/clause_6_overview.adoc (hunk 2 above)
Comment (jerstlouis) [reply core/standard/clause_6_overview.adoc:183]: Yes, thank you, I think we should review this one with Joan on Thursday if we can meet.

**Comment by jerstlouis** [comment]
Merged in master.

**Pull request closed without merging**